mmv1/products/compute/Disk.yaml

```diff
@@ -511,3 +511,16 @@ properties:
       * /projects/{project}/zones/{zone}/storagePools/{storagePool}
     diff_suppress_func: 'tpgresource.CompareResourceNames'
     custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+  - !ruby/object:Api::Type::String
+    name: 'accessMode'
+    required: false
+    immutable: false
+    description: |
+      The accessMode of the disk.
+      For example:
+      * READ_WRITE_SINGLE
+      * READ_WRITE_MANY
+      * READ_ONLY_SINGLE
+    default_from_api: true
```

Comment:
What is the reason to use `default_from_api: true` here? `default_from_api: true` is used when the field has a default value set the service.

Reply:
I was referring to enabledConfidentialCompute field https://github.com/chentaozgoogle/magic-modules/blob/main/mmv1/products/compute/Disk.yaml#L438 . They are similar case in regard with default value, e.g. some disk doesn't have the field at all, some disk has the field, and when the disk has  the field, the default is determine by the backend API server.   I though `default_from_api: true` is for the case. Am I wrong? if `default_from_api: tru` field is not meant for that, I can remove it.

Reply:
It meant that if the value of the field is not provided by the user, the API server will set a default value, then the default value is sent back to the API in the following requests if no provided value by the user. If a value is provided by the user, the user's value is sent to API.

When no value is provided by the user,  the server API will set a default value for it. Then `default_from_api: true` can be used. 

When no value is provided by the user,  will the server API set a default value for it?

Reply:
Some disk may not have field, so the API server will not return the field at all.
For disks that do have the field,  when the no value is provided by the user, the API server will return the default value.